library/src/webapp/js/spinner.js

```diff
@@ -353,7 +356,7 @@ SPNR.disableElementAndSpin = function( divID, element, activateSpinner )
         // <button>               --> localName = "button", nodeName="button", tagName="BUTTON", type="submit"
         // <button type="button"> --> localName = "button", nodeName="button", tagName="BUTTON", type="button"
         element.tagName === "BUTTON" ? newElement.innerHTML = element.innerHTML : newElement.textContent = element.getAttribute( "value" );
-        newElement.className = activateSpinner ? "spinButton formButtonDisabled" : "formButtonDisabled";
+        newElement.className = element.className + " " + (activateSpinner ? "btn spinButton formButtonDisabled" : "btn formButtonDisabled");
```

Comment:
I'm not sure why the "btn" class is necessary but I'm assuming you've determined it to be so.

Reply:
Without "btn", it won't clone the element with the same style and spinner

Reply:
Maybe something like this:

```suggestion
    if( escapeList === null )
    {
        escapeList = [];
        escapeList = [
            "current-site-button",
            "sidebar-collapse-button"
        ];
    }
    else
    {
        if( !escapeList.includes("current-site-button") )
        {
            escapeList.push( "current-site-button" );
        }
        if( !escapeList.includes("sidebar-collapse-button") )
        {
            escapeList.push( "sidebar-collapse-button" );
        }
    }
```

I'm not very up-to-date on my JS, there may be a more elegant approach to do this.